packages/vector_graphics/lib/src/render_vector_graphics.dart

```diff
@@ -130,11 +147,11 @@ class RenderVectorGraphic extends RenderBox {
 
   @override
   void paint(PaintingContext context, ui.Offset offset) {
+    assert(size == pictureInfo.size);
```

Comment:
Because we always put this RO in a sized box with the size of the picture info, this must be true. Therefore, we also cannot compute any scaling information here - since both target and desired size will be identical